packages/mc-scripts/README.md

```diff
@@ -4,10 +4,22 @@
   <a href="https://www.npmjs.com/package/@commercetools-frontend/mc-scripts"><img src="https://badgen.net/npm/v/@commercetools-frontend/mc-scripts" alt="Latest release (latest dist-tag)" /></a> <a href="https://www.npmjs.com/package/@commercetools-frontend/mc-scripts"><img src="https://badgen.net/npm/v/@commercetools-frontend/mc-scripts/next" alt="Latest release (next dist-tag)" /></a> <a href="https://bundlephobia.com/result?p=@commercetools-frontend/mc-scripts"><img src="https://badgen.net/bundlephobia/minzip/@commercetools-frontend/mc-scripts" alt="Minified + GZipped size" /></a> <a href="https://github.com/commercetools/merchant-center-application-kit/blob/master/LICENSE"><img src="https://badgen.net/github/license/commercetools/merchant-center-application-kit" alt="GitHub license" /></a>
 </p>
 
-Configuration and scripts for developing a MC application.
+Configuration and scripts for developing a Custom Application.
 
 ## Install
 
 ```bash
 $ npm install --save @commercetools-frontend/mc-scripts
 ```
+
+## Usage
+
+The package exports some functions to configure Webpack
+
+```js
+const {
+  createWebpackConfigForDevelopment,
+  createWebpackConfigForProduction,
+  vendorsToCompile,
+} = require('@commercetools-frontend/mc-scripts');
```

Comment:
This is new, but there is proxy `config` folder for backwards compatibility.

Reply:
Ah, thanks. Disregard my comment regarding that in the review then.